syft/frameworks/torch/he/fv/util/operations.py

```diff
@@ -111,22 +152,26 @@ def poly_mul_mod(op1, op2, coeff_mod):
         A list with polynomial coefficients.
     """
 
-    # For non same size polynomails we have to shift the polynomials because numpy consider right
+    # For non same size polynomials we have to shift the polynomials because numpy consider right
     # side as lower order of polynomial and we consider right side as heigher order.
-    if len(op1) != len(op2):
-        if len(op1) > len(op2):
-            op2 = op2 + [0] * (len(op1) - len(op2))
-        else:
-            op1 = op1 + [0] * (len(op2) - len(op1))
+    if len(op1) != poly_mod:
+        op1 += [0] * (poly_mod - len(op1))
+    if len(op2) != poly_mod:
+        op2 += [0] * (poly_mod - len(op2))
 
-    poly_mod = np.array([1] + [0] * (len(op1) - 1) + [1])
+    poly_len = poly_mod
```

Comment:
might be good to refactor the name of poly_mod to only when we have a polynomial, and use poly_len for the length of the polynomial, just like in this function, but in the whole BFV implementation